packages/api/src/server.ts

```diff
@@ -1,104 +1,93 @@
 import { readFileSync } from "node:fs";
 import { resolve } from "node:path";
-import bodyParser from "body-parser";
-import compression from "compression";
-import express, { Request, Response, NextFunction, Application } from "express";
+import fastify, { FastifyInstance } from "fastify";
 import RpcError from "types/lib/api/errors/rpc-error";
-import ApplicationError from "types/lib/api/errors/application-error";
 import logger from "./logger";
 
 const packageJson = resolve(process.cwd(), "package.json");
 
-function logResponseTime(
-  req: Request,
-  res: Response,
-  next: NextFunction
-): void {
-  const startHrTime = process.hrtime();
-
-  res.on("finish", () => {
-    const elapsedHrTime = process.hrtime(startHrTime);
-    const elapsedTimeInMs = elapsedHrTime[0] * 1000 + elapsedHrTime[1] / 1e6;
-    let message = `${req.method} ${res.statusCode} ${elapsedTimeInMs}ms\t${req.path}`;
-    if (req.method === "POST") {
-      message += JSON.stringify(req.body);
-    }
-    logger.log({
-      level: "info",
-      message,
-      consoleLoggerOptions: { label: "API" },
-    });
-  });
-
-  next();
-}
-
 export class Server {
-  private app: Application;
+  private app: FastifyInstance;
 
   constructor() {
-    this.app = express();
+    this.app = fastify({
+      logger,
+      disableRequestLogging: true,
```

Comment:
Can this be configurable? Will be useful while testing/monitoring hosted instances, no?